libs/SmartStore/SmartStoreTests/SFSmartStoreFullTextSearchTests.h

```diff
@@ -27,5 +27,12 @@
 
 #import "SFSmartStoreTestCase.h"
 
-@interface SFSmartStoreFullTextSearchTests : SFSmartStoreTestCase 
+@interface SFSmartStoreFullTextSearchTests : SFSmartStoreTestCase
+
+#define kTestStore            @"testSmartStoreFullTextSearchStore"
```

Comment:
I have only be using #define in tests (and usually in the .m). I agree we should use const in the code. For tests, I don't think it matters.